Jenkinsfile.kola.gcp

```diff
@@ -41,7 +41,7 @@ def pod_label = "cosa-${UUID.randomUUID().toString()}"
 
 podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
     node(pod_label) { container('coreos-assembler') {
-        def gcp_image, no_gcp_image
+        def gcp_image
         def no_gcp_image = false
```

Comment:
I think it'd be cleaner to just get rid of `no_gcp_image` entirely and initialize `gcp_image` to `""` and fail on that.

Reply:
ok I updated it but I need you to help me to make sure the conditional logic for `""` is correct